NAMESPACE

```diff
@@ -4,6 +4,8 @@ S3method(MSstatsContrastMatrix,character)
 S3method(MSstatsContrastMatrix,data.frame)
 S3method(MSstatsContrastMatrix,list)
 S3method(MSstatsContrastMatrix,matrix)
+export(.get.plotly.plot.html)
```

Comment:
can't we use the same naming convention as usually?
Is this function meant to be used by users directly or by the shiny app?

Reply:
Commit [fix name and exports](https://github.com/Vitek-Lab/MSstats/pull/107/commits/5d88fc0353c4a1e3ae1add4b068eec4a91ef93da)